.ci/cico_updates_minishift.sh

```diff
@@ -65,10 +65,6 @@ installLatestCheStable() {
   # Add stable Che images and tag to CR
   sed -i "s/cheImage: ''/cheImage: quay.io\/eclipse\/che-server/" ${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml
   sed -i "s/cheImageTag: ''/cheImageTag: ${previousPackageVersion}/" ${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml
-  sed -i "s/devfileRegistryImage: ''/devfileRegistryImage: quay.io\/eclipse\/che-devfile-registry:"${previousPackageVersion}"/" ${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml
-  sed -i "s/pluginRegistryImage: ''/pluginRegistryImage: quay.io\/eclipse\/che-plugin-registry:"${previousPackageVersion}"/" ${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml
-  sed -i "s/identityProviderImage: ''/identityProviderImage: quay.io\/eclipse\/che-keycloak:"${previousPackageVersion}"/" ${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml
-
```

Comment:
Why we remove this images?

Reply:
chectl doesn't set this images in the CR anymore, so we don't have to update them.

Reply:
@flacatus and @tolusha we really don't need to set up these images in the CR anymore, because values are empty.  But chectl reuse nightly templates, where is in the operator.yaml was set up images defaults with nightly tags. So I fixed this stuff, I replaced images defaults using `sed` and I applied --template argument to use current che-operator  templates in the chectl. Let me know, if you don't like this solution.